src/settings/CustomFieldsSettings.js

```diff
@@ -16,8 +17,14 @@ const CustomFieldsSettings = ({
 }) => {
   const stripes = useStripes();
 
+  const base = '/settings/users/custom-fields';
+
   const redirectToEdit = () => {
-    history.push('/users/custom-fields/edit');
+    history.push(`${base}/edit`);
+  };
+
+  const redirectToView = () => {
+    history.push(`${base}`);
   };
```

Comment:
These get rendered as `<Buttons>` in SSC; can't we just use the `to` prop and let `react-router` take care of history manipulation?

Reply:
Additional PR for this change: https://github.com/folio-org/stripes-smart-components/pull/778